File: VB/WebSite/App_Code/LiveDataItem.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports DevExpress.Web.ASPxGridView
Imports DevExpress.Web.ASPxEditors

Public Class LiveDataItem
	Implements ITemplate
	Public Sub New()
	End Sub

	Public Sub InstantiateIn(ByVal container As Control) Implements ITemplate.InstantiateIn
		Dim gridContainer As GridViewDataItemTemplateContainer = TryCast(container, GridViewDataItemTemplateContainer)

		Dim label As New ASPxLabel()
		label.ID = "lbText"
		gridContainer.Controls.Add(label)

		label.Width = New Unit(100, UnitType.Percentage)
		label.ClientInstanceName = String.Format("lbLiveText_{0}_{1}", gridContainer.VisibleIndex, gridContainer.Column.Index)

		label.Value = DataBinder.Eval(gridContainer.DataItem, gridContainer.Column.FieldName)
	End Sub
End Class

File: CS/WebSite/App_Code/LiveDataItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using DevExpress.Web.ASPxGridView;
using DevExpress.Web.ASPxEditors;

public class LiveDataItem : ITemplate {
    public LiveDataItem() {
    }

    public void InstantiateIn(Control container) {
        GridViewDataItemTemplateContainer gridContainer = container as GridViewDataItemTemplateContainer;

        ASPxLabel label = new ASPxLabel();
        label.ID = "lbText";
        gridContainer.Controls.Add(label);

        label.Width = new Unit(100, UnitType.Percentage);
        label.ClientInstanceName = String.Format("lbLiveText_{0}_{1}", gridContainer.VisibleIndex, gridContainer.Column.Index);

        label.Value = DataBinder.Eval(gridContainer.DataItem, gridContainer.Column.FieldName);
    }
}